utils/load_unitconv.m

```diff
@@ -59,7 +88,7 @@ function load_unitconv(ringmode, renamedIndexes)
         hcor_index = hcor_index + 1;
     end
     id = write_linear_data(data.field(2) / data.current(2), 0);
-    fprintf(f_units, '%d,%s,poly,%d\n', renamedIndexes(hcor_index), 'x_kick', id); 
+    fprintf(f_units, '%d,%s,poly,%d,%s,%s\n', renamedIndexes(hcor_index), 'x_kick', id, 'rads^-2', 'A'); 
```

Comment:
Not sure where `rads^-2` came from. The value is in radians, but for simplicity we are treating radians as dimensionless: https://en.wikipedia.org/wiki/Radian#Dimensional_analysis

Reply:
In that case, should it be 'rads', 'A' or '', 'A'?

Reply:
'', 'A'